doc/latex.rst

```diff
@@ -186,16 +187,50 @@ Keys that you may want to override include:
    .. versionchanged:: 2.0
       Incorporates some font substitution commands to help support occasional
       Greek or Cyrillic in a document using ``'pdflatex'`` engine.
+      At 4.0.0 these commands were moved to the ``'fontsubstitution'`` key.
 
    .. versionchanged:: 4.0.0
 
-      - The font substitution commands added at ``2.0`` have been moved
-        to the ``'fontsubstitution'`` key, as their presence here made
-        it complicated for user to customize the value of ``'fontpkg'``.
-      - The default font setting has changed: it still uses Times and
-        Helvetica clones for serif and sans serif, but via better, more
-        complete TeX fonts and associated LaTeX packages.  The
-        monospace font has been changed to better match the Times clone.
+      The default font setting was changed.  As shown above it still uses
+      Times and Helvetica clones for serif and sans serif, but via better,
+      more complete TeX fonts and associated LaTeX packages.  The monospace
+      font has been changed to better match the Times clone.
+
+   .. versionchanged:: 8.1.0 Configuration of math fonts should be done via
+      the ``'fontpkgmath'`` key, which is new at 8.1.0.  For ``'pdflatex'``
+      this key injects ``\usepackage{amssymb}`` which is thus now executed
+      after the contents of ``'fontpkg'``.
+
+``'fontpkgmath'``
+   LaTeX package for math fonts.  It defaults to:
+
+   - ``r'\usepackage{amssymb}'`` if :confval:`latex_engine` is set to
+     ``'pdflatex'`` (or ``'uplatex'``).
+
+   - ``r'\usepackage{unicode-math}'`` for ``'xelatex'`` and ``'lualatex'``
+     with extras to set the math font to XITS Math (which is an excellent
+     match to FreeSerif text font).  It also makes mark-up such as
```

Comment:
Actually in my testing XITS Math digits and upright Latin letters are quasi-identical with those of FreeSerif (I can not discern any difference visually).

![Capture d’écran 2024-08-10 à 09 05 23](https://github.com/user-attachments/assets/f23454a0-51f6-4c13-87d6-ecc8214d3215)

(character boxes widths may differ slighly for example for letter f and kerning differs due to XITS Math being  a math font not a text font).

Reply:
Now, upright XITS Math is used only in operator names, and having exactly identical glyphs with text is not extremely important, what is important is that the weight of the XITS Math glyphs for example for Greek is much better fitted to text.

Compare with this PR:
![Capture d’écran 2024-08-10 à 09 18 21](https://github.com/user-attachments/assets/f133db44-2a13-4108-bf89-e81581a1e25a)

and without this PR:
![Capture d’écran 2024-08-10 à 09 18 01](https://github.com/user-attachments/assets/b5b6fe2a-2398-4620-ad63-20250317e70c)